hw/top_darjeeling/templates/toplevel.sv.tpl

```diff
@@ -388,7 +397,10 @@ for rst in output_rsts:
     cg_en = 'cg_' + lpg['clock_connection'].split('clk_')[-1]
   else:
     cg_en = top['clocks'].hier_paths['lpg'] + lpg['clock_connection'].split('.clk_')[-1]
-  rst_en = lib.get_reset_lpg_path(top, lpg['reset_connection'])
+  if lpg['unmanaged_reset']:
+    rst_en =  top['unmanaged_resets']._asdict()[lpg['reset_connection']['name']].rst_en_signal_name
```

Comment:
Would it make sense to define something like this in `UnmanagedResets`?
```python
    def get(self, name: str) -> object:
        try:
            return self.resets['name']
        except KeyError:
            raise ValueError(f"No reset defined with name {name}") from None
```
It might be a way to push information out of this file and give better error messages if something doesn't get connected up properly.